bcgov/moh-prime
# PRIME 2174 display user type on next steps and provisioner link

To test:

1. create an enrolment
2. accept toa
3. on next steps page submit email to send link to
4. The alert should reference what the user is (OBO/Independent User)
5. Go to mailhog
6. follow provisioner link
7. page should show user type
